okhttp-sse/src/main/kotlin/okhttp3/sse/EventSourceListener.kt

```diff
@@ -18,6 +18,11 @@ package okhttp3.sse
 import okhttp3.Response
 
 abstract class EventSourceListener {
+  /**
+   * Seconds elapsed between 2 events until connection failed. Doesn't timeout if null
+   */
+  open var timeout: Long? = null
```

Comment:
From Call, I think there is a convention on timeout names.  Such as `readTimeoutMillis`. So perhaps `receiveTimeoutMillis`? 

I wonder if we should prefer Kotlin Duration here also?

Reply:
[`callTimeoutMillis`](https://github.com/square/okhttp/blob/02d69bf883c47905d0ad3fd9d2ad75b8ede75bb9/okhttp/src/main/kotlin/okhttp3/OkHttpClient.kt#L236) ?